packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1207,7 +1216,7 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
       _imageStream!.removeListener(_getListener());
 
     if (!widget.gaplessPlayback)
-      setState(() { _imageInfo = null; });
+      setState(() => _replaceImage(info: null));
```

Comment:
avoid => in setState since it doesn't return anything

Reply:
Done